internal/core/thirdparty/knowhere/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@
 # or implied. See the License for the specific language governing permissions and limitations under the License.
 #-------------------------------------------------------------------------------
 
-set( KNOWHERE_VERSION v2.1.0 )
+set( KNOWHERE_VERSION main )
```

Comment:
Is this by design to follow Knowhere main branch?

Reply:
by design